vochain/process_id.go

```diff
@@ -0,0 +1,120 @@
+package vochain
+
+import (
+	"encoding/binary"
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+	"go.vocdoni.io/dvote/crypto/ethereum"
+	"go.vocdoni.io/proto/build/go/models"
+	"google.golang.org/protobuf/encoding/protowire"
+)
+
+// ProcessID is a 32 bytes identifier that holds the following information about the voting process:
+//
+// - chainID: a 6 bytes trunked hash of the blockchain identifier
+//
+// - organization: hex address of the wallet creating the process
+//
+// - census proof type: kind of census used for the process (from protobuf models)
+//
+// - nonce: an incremental uint32 number
+//
+// processID: 395b41cc9abcd8da6bf26964af9d7eed9e03e53415d37aa96045000600000001 represents
+// chID:vocdoni-azeno addr:0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045 pt:arbo nc:1
+type ProcessID struct {
+	chainID          string
+	organizationAddr *common.Address
+	proofType        uint16
+	nonce            uint32
+}
+
+// Marshal encodes to bytes
+func (p *ProcessID) Marshal() []byte {
```

Comment:
might as well implement https://pkg.go.dev/encoding#BinaryMarshaler and its unmarshaler counterpart